public/locale/en.json

```diff
@@ -64,7 +64,7 @@
   "DOMESTIC_HEALTHCARE_SUPPORT__PAID_CAREGIVER": "Paid caregiver",
   "GENDER__1": "Male",
   "GENDER__2": "Female",
-  "GENDER__3": "Non-binary",
+  "GENDER__3": "Transgender",
```

Comment:
This reminds me of something I came across while testing the user redesign (forgot about it 😅 ) - Back end designates 3 as Non-binary, so we need to update it there as well.

Reply:
Lets be in sync with the BE